docs/voicemail/how-to/index.md

```diff
@@ -18,39 +18,43 @@ To list and play voicemails you will need to:
 
 ### Install dependencies
 
-Add the dependency to your `pom.xml`:
+To add the dependencies, first you need to add the [Jitpack](https://jitpack.io) repository:
 ```xml
-<!-- Add jitpack repository -->
-//...
-    <repositories>
-        <repository>
-            <id>jitpack.io</id>
-            <url>https://jitpack.io</url>
-        </repository>
-    </repositories>
-</project>
-
-
-<!-- Add dependency for voicemail-grpc and common -->
-<dependency>
-    <groupId>com.github.working-group-two.wgtwoapis</groupId>
-    <artifactId>voicemail-grpc</artifactId>
-    <version>master</version>
-</dependency>
-<dependency>
-    <groupId>com.github.working-group-two.wgtwoapis</groupId>
-    <artifactId>common</artifactId>
-    <version>master</version>
-</dependency>
+<repositories>
+    <repository>
+        <id>jitpack.io</id>
+        <url>https://jitpack.io</url>
+    </repository>
+</repositories>
 ```
 
+Then you can add `voicemail-grpc` and `common`:
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.github.working-group-two.wgtwoapis</groupId>
+        <artifactId>voicemail-grpc</artifactId>
+        <version>master</version>
+    </dependency>
+    <dependency>
+        <groupId>com.github.working-group-two.wgtwoapis</groupId>
+        <artifactId>common</artifactId>
+        <version>master</version>
+    </dependency>
+</dependencies>
+```
+
+Setting the version to `master` means that Jitpack will build whatever is on the master branch when the dependency is resolved.
+
 ### Initialize your dependencies
 ```kotlin
 import com.wgtwo.api.auth.Clients
 import com.wgtwo.api.common.OperatorToken
+import com.wgtwo.example.Secrets
 
-val channel = Clients.createChannel(Clients.Environment.PROD)
-val credentials = OperatorToken("acb...", "xyz...") // or UserToken("abc123...")
+val channel = ManagedChannelBuilder.forAddress("api.wgtwo.com", 443).build()
+val credentials = BasicAuth("YOUR_CLIENT_ID", "YOUR_CLIENT_SECRET")
```

Comment:
The `"YOUR_CLIENT_ID", "YOUR_CLIENT_SECRET"` change is fine. Is the rest intentional?